kotlin-react/src/main/kotlin/react/RComponent.kt

```diff
@@ -16,10 +16,6 @@ abstract class RComponent<P : RProps, S : State> : Component<P, S> {
     // if you use this method, don't forget to pass props to the constructor first
     open fun S.init(props: P) {}
 
-    fun RBuilder.children() {
```

Comment:
Redundant?

Reply:
Yea, there is `context(RBuilder) Props.children()`, this declaration in more obvious, it is impossible to declare such method in `RComponent` because we don't know will inheritor implements `PropsWithChildren` or not